<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editScheduleDialog</class>
 <widget class="QDialog" name="editScheduleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(253, 250, 246);</string>
  </property>
  <widget class="QLineEdit" name="taskNmLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgb(228, 239, 231);
	border-radius: 5px;
}
QLineEdit::focus {
	border: 1px solid rgb(214, 225, 217);  
}</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="memoTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	background-color: rgb(228, 239, 231);
	border-radius: 5px;
}
QTextEdit::focus {
	border: 1px solid rgb(214, 225, 217);  
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgb(228, 239, 231);
	border-radius: 5px;
}
QLineEdit::focus {
	border: 1px solid rgb(214, 225, 217);  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>메모</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Task</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>장소</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>시간</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="startDateTimeEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 239, 231);
border-radius: 5px;</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>4</month>
     <day>14</day>
    </datetime>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2025</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd hh:mm</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="endDateTimeEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 239, 231);
border-radius: 5px;</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>4</month>
     <day>14</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2025</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::Section::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd hh:mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>부터</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>까지</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(153, 188, 133);
	border: 1px solid #ccc;
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(139, 171, 121);
}</string>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid #ccc;
	background-color: rgb(250, 241, 230);
	border-radius: 5px;
}
QPushButton:hover {
	background-color:  rgb(236, 228, 217);
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>17</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(153, 188, 133);
border-radius: 5px;
</string>
        </property>
        <property name="midLineWidth">
         <number>-1</number>
        </property>
        <property name="text">
         <string>일정 추가</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="currentDate">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>17</height>
       </size>
      </property>
      <property name="text">
       <string>0000년 00월 00일</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>101</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="repeat_check">
      <property name="text">
       <string>반복</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="repeat_combo">
      <property name="styleSheet">
       <string notr="true">QComboBox{
	background-color: rgb(228, 239, 231);
	border-radius: 5px;
}
QComboBox::focus {
	border: 1px solid rgb(214, 225, 217);  
}</string>
      </property>
      <item>
       <property name="text">
        <string>반복 안 함</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1주 마다</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1개월 마다</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
